opencensus/trace/blank_span.py

```diff
@@ -15,9 +15,10 @@
 from opencensus.trace import time_event as time_event_module
 from opencensus.trace.span_context import generate_span_id
 from opencensus.trace.tracers import base
+from opencensus.trace.base_span import BaseSpan
```

Comment:
Sort the imports, and import module instead of class.

Reply:
Done